tests/Stl.Tests/Interception/MethodBasedFactoryTest.cs

```diff
@@ -0,0 +1,32 @@
+using Stl.Interception;
+
+namespace Stl.Tests.Interception;
+
+public class MethodBasedFactoryTest
+{
+    [Fact]
+    public void TestMethodBasedFactory()
```

Comment:
Can't test locally, as I don't have any .NET 8 RC installed.

Reply:
Interesting :)

Reply:
I'll check it out :)

Reply:
So, more info on that:
```
dotnet test -f net7.0```
Doesn't work.

But
```
dotnet restore; dotnet build -f net7.0 --no-restore
```
Does. Weird.